tests/basic.spec.ts

```diff
@@ -44,7 +44,6 @@ test('popover as "manual"', async ({ page }) => {
 
 test('popover as "invalid"', async ({ page }) => {
   const popover = (await page.locator('#popover6')).nth(0);
-  await expect(popover).toBeVisible();
```

Comment:
This expectation has been dropped as it does not hold true in a default Chrome environment.